src/components/graph/graph.helper.js

```diff
@@ -451,11 +451,23 @@ function updateNodeHighlightedValue(nodes, links, config, id, value = false) {
     };
 }
 
+/**
+ *
```

Comment:
Please add a simple description (I think it might even be required for linting purposes).